### DISCUSSION TITLE
Why init `ACE_Synch_Options` by `ACE_Time_Value()` rather than `ACE_Time_Value::zero`

### DISCUSSION THREAD

**Discussion by likema** [DOCGroup/ACE_TAO#2178] [Q&A] [answered]
Why https://github.com/DOCGroup/ACE_TAO/commit/56b5f840e64 revises `ACE_Synch_Options` CTOR argument `ACE_Time_Value::zero` to `ACE_Time_Value()` ?

**Comment by mitza-oci** [comment, marked as answer]
The PR that this commit came from https://github.com/DOCGroup/ACE_TAO/pull/1879 has a description of why it changed.  Is that sufficient?

**Reply by likema** [reply]
Got it. Thanks
